backend/models/group.js

```diff
@@ -152,6 +154,8 @@ groupSchema.statics.ifcGuidsToUUIDs = function (account, model, ifcGuids, branch
 groupSchema.statics.uuidToIfcGuids = function (obj) {
 	const account = obj.account;
 	const model = obj.model;
+
+	// @ts-ignore
```

Comment:
ts-ignore?! backend uses eslint, i don't think this needs to be here? doesn't trigger linter on my local repo.

Reply:
I think is because I have something wrong in my visual studio condifguration. Ill get rid of it.